src/fhir-tool/Program.cs

```diff
@@ -731,44 +733,39 @@ private static IList<Questionnaire> GetQuestionnairesFromFlatFileFormatV1(string
-                };
+                Questionnaire questionnaire = new Questionnaire();
 
                 questionnaire.Meta = new Meta
                 {
-                    Profile = new string[] { "http://ehelse.no/fhir/StructureDefinition/sdf-questionnaire" },
-                    Tag = new List<Coding>
-                    {
-                        // TODO: Vi trenger definere en Tag som anigr det tekniske navnet til
-                        //new Coding("http://fhi.no/fylkeshelseundersokelse", "1.0"),
-                        //new Coding("http://helfo.no/questionnaire", "E121"),
-                        
-                        // TODO: Vi trenger definere tilgangsstyring i Excel-arket
-                        new Coding("urn:2.16.578.1.12.4.1.1.7607", "3")
-                    }
+                    Profile = new string[] { "http://ehelse.no/fhir/StructureDefinition/sdf-Questionnaire" }
                 };
 
+                if (!string.IsNullOrEmpty(masterDetail.Master.Id))
+                    questionnaire.Id = masterDetail.Master.Id;
+                if (!string.IsNullOrEmpty(masterDetail.Master.Url))
+                    questionnaire.Url = masterDetail.Master.Url;
+                if (!string.IsNullOrEmpty(masterDetail.Master.Version))
+                    questionnaire.Version = masterDetail.Master.Version;
+                if (!string.IsNullOrEmpty(masterDetail.Master.Name))
+                {
+                    // TODO: Name er søkbart, vi må definere dette som en unik del enten kun ved seg selv eller i kombinasjon med eksempelvis en Tag
+                    questionnaire.Name = masterDetail.Master.Name;
+                }
+                if (!string.IsNullOrEmpty(masterDetail.Master.Title))
+                    questionnaire.Title = masterDetail.Master.Title;
+                if (!string.IsNullOrEmpty(masterDetail.Master.Status))
+                    questionnaire.Status = EnumUtility.ParseLiteral<PublicationStatus>(masterDetail.Master.Status);
+                if (!string.IsNullOrEmpty(masterDetail.Master.Date))
+                    questionnaire.Date = masterDetail.Master.Date;
+                if (!string.IsNullOrEmpty(masterDetail.Master.Publisher))
+                    questionnaire.Publisher = masterDetail.Master.Publisher;
+                if (!string.IsNullOrEmpty(masterDetail.Master.Description))
+                    questionnaire.Description = new Markdown(masterDetail.Master.Description);
+                if (!string.IsNullOrEmpty(masterDetail.Master.Purpose))
+                    questionnaire.Purpose = string.IsNullOrEmpty(masterDetail.Master.Purpose) ? null : new Markdown(masterDetail.Master.Purpose);
+                if (!string.IsNullOrEmpty(masterDetail.Master.Contact))
+                    questionnaire.Contact = new List<ContactDetail> { new ContactDetail { Name = masterDetail.Master.Contact } };
+                
```

Comment:
This has changed from previous version. Was:
Contact = new List<ContactDetail> { new ContactDetail { Telecom = new List<ContactPoint> { new ContactPoint { System = ContactPoint.ContactPointSystem.Url, Value = masterDetail.Master.Contact } } } }
Is the new version correct?